include/aws/compression/exports.h

```diff
@@ -15,14 +15,13 @@
 #        define AWS_COMPRESSION_API
 #    endif /* AWS_COMPRESSION_USE_IMPORT_EXPORT */
 
-#else /* defined (USE_WINDOWS_DLL_SEMANTICS) || defined (WIN32) */
-#    if ((__GNUC__ >= 4) || defined(__clang__)) && defined(AWS_COMPRESSION_USE_IMPORT_EXPORT) &&                       \
-        defined(AWS_COMPRESSION_EXPORTS)
+#else /* defined (AWSC_RT_USE_WINDOWS_DLL_SEMANTICS) || defined (WIN32) */
```

Comment:
comment has rotten (new AWS_CRT_USE_WINDOWS_DLL_SEMANTICS vs old AWSC_RT_USE_WINDOWS_DLL_SEMANTICS)

Reply:
fixed the typo